Sem. 01/task6.cpp

```diff
@@ -0,0 +1,92 @@
+#include <iostream>
+
+using std::cout;
+using std::cin;
+using std::endl;
+
+struct Point
+{
+	double x, y;
+};
+
+void GetPoint(Point& p)
+{
+	cin >> p.x >> p.y;
+}
+
+void PrintPoint(Point& p)
+{
+	cout << "(" << p.x << ";" << p.y << ")";
+}
+
+double GetDist(Point& p1, Point& p2)
+{
+	double dx = p2.x - p1.x;
+	double dy = p2.y - p1.y;
+	return sqrt((dx * dx) + (dy * dy));
+}
+
+double GetDistFromCenter(Point& p1)
+{
+	double dx = 0 - p1.x;
+	double dy = 0 - p1.y;
+	return sqrt((dx * dx) + (dy * dy));
+}
+
+void whichQuadrant(Point& p)
```

Comment:
Be consistent when you name your functions, in the other you used PascalCase naming